src/components/flow/routers/subflow/SubflowRouterForm.test.ts

```diff
@@ -1,56 +1,65 @@
 import SubflowRouterForm from 'components/flow/routers/subflow/SubflowRouterForm';
+import { FlowTypes } from 'config/interfaces';
 import { composeComponentTestUtils, mock, setMock } from 'testUtils';
 import {
   ColorFlowAsset,
+  SoundFlowAsset,
   createStartFlowAction,
+  createVoiceStartFlowAction,
   createSubflowNode,
   getRouterFormProps
 } from 'testUtils/assetCreators';
 import * as utils from 'utils';
 
-const routerNode = createSubflowNode(createStartFlowAction());
-const { setup } = composeComponentTestUtils(SubflowRouterForm, getRouterFormProps(routerNode));
+const msgRouterNode = createSubflowNode(createStartFlowAction(), FlowTypes.MESSAGING);
+const { setup: msgSetup } = composeComponentTestUtils(
+  SubflowRouterForm,
+  getRouterFormProps(msgRouterNode)
+);
 mock(utils, 'createUUID', utils.seededUUIDs());
 
 describe(SubflowRouterForm.name, () => {
-  describe('render', () => {
+  describe('msg render', () => {
     it('should render', () => {
-      const { wrapper } = setup();
+      const { wrapper } = msgSetup();
       expect(wrapper).toMatchSnapshot();
     });
   });
 
-  describe('updates', () => {
+  describe('msg updates', () => {
     it('should update and save', () => {
-      const { instance, props } = setup(true, { updateRouter: setMock() });
+      const { instance, props } = msgSetup(true, { updateRouter: setMock() });
+      expect(instance.context.config.flowType).toEqual(FlowTypes.MESSAGING);
```

Comment:
@ericnewcomer i added a line to each test to explicitly check the flowType, but when i uncomment the ivr tests, the msg ones suddenly fail, i'm not sure why, but when i step through, it almost seems as though something like this is happening:
- msg router node and setup are created
- ivr router node and setup are created (but it almost seems like this might be overwriting the msg stuff?)
- msg tests are run (everything passes except for the flow type assertions)
- ivr tests are run (everything passes)
any ideas as to how i can resolve this? i'm hoping maybe it's just something quirky with my test setup?